unicodetools/data/ucd/dev/PropertyAliases.txt

```diff
@@ -73,6 +73,7 @@ EqUIdeo                  ; Equivalent_Unified_Ideograph
 FC_NFKC                  ; FC_NFKC_Closure
 lc                       ; Lowercase_Mapping
 NFKC_CF                  ; NFKC_Casefold
+NFKC_SCF                 ; NFKC_SimpleCasefold
```

Comment:
This name looks very weird in having both Low_Lines and CamelCase, whereas the non-Unihan properties separate words by Low_Lines (they treat casefold as one word but case folding as two, but that is a different question).

Should this be NFKC_Simple_Casefold? (This would require the UTC to change its mind, but there is still time for that).

Reply:
unicode-org/properties#110.